src/client/client.c

```diff
@@ -1,9 +1,10 @@
-#include "client.h"
 #include "libhirte/bus/utils.h"
-#include <stdio.h>
+#include "libhirte/log/log.h"
+
+#include "client.h"
 
 int list_units_on_all(UNUSED Client *client) {
-        printf("Listing units on all\n");
+        hirte_log_info("Listing units on all");
```

Comment:
hm, are we sure about this? This is the client, so we want it to print things to stdout, not log them somewhere

Reply:
Good point! I removed the logging from the client for now.